.github/actions/pylint/README.md

```diff
@@ -30,3 +30,14 @@ with:
   pylint-rcfile: "./pyproject.toml"
   pylint-paths: "python tests"
 ```
+
+## Building
+
+To bundle the source for use as a GitHub Action, assuming the current
```

Comment:
[question] Right now, the build is in the source tree and checked in (which is less common for us). Is this going to change when we release it properly, or does all that still have to happen? If so, be good to explain that in a little more detail - as it might be less common for people with the general background of the project (python/cpp).